### PR TITLE
Add exam mailer

### PR DESCRIPTION
**Pull request by vuducdat**

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```
Comment (leminhtuan2015) [inline app/mailers/application_mailer.rb]: from@example.com Should be placed in config.

File: config/environments/development.rb [hunk 2]
```diff
@@ -38,5 +38,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
-  config.action_mailer.default_url_options = {host: "localhost", port: 3000}
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :test
+  host = "localhost:3000"
```
Comment (leminhtuan2015) [inline config/environments/development.rb]: Config please.

File: app/views/exam_mailer/exam_notification.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1><%= t "messages.title" %></h1>
+
+<p>
+  <%= t "messages.greeting" %> <%= @user.name %>,
```
Comment (thanhnga67) [inline app/views/exam_mailer/exam_notification.html.erb]: , ? and convention space

**Comment by biznezo** [comment]
OK

**Comment by thanhnga67** [comment]
LGTM

**Pull request merged**